### ISSUE TITLE
boophone

### ISSUE THREAD

**Issue by ROBERT-MCDOWELL** [ruffle-rs/ruffle#7087]
### Describe the bug

from https://secure.boophone.com

when ruffle loaded the swf appears well but on the console:
attached log
[secure.boophone.com-1653488961162.log](https://github.com/ruffle-rs/ruffle/files/8772026/secure.boophone.com-1653488961162.log)

### Expected behavior

no JS error

### Affected platform

Self-hosted version

### Operating system

ALL

### Browser

ALL

**Comment by torokati44** [comment]
This movie contains a `VideoStream` with `codecID = 0`, which means that there is no actual video data (`VideoFrame`) in the SWF, instead, the video data comes from an external source, usually through some kind of a network stream - which is not yet supported by Ruffle.

**Comment by ROBERT-MCDOWELL** [comment]
Yes I know that streaming is not yet supported. I thought the JS errors were related to fonts or something like that

**Comment by torokati44** [comment]
I see no mention of fonts in the attached error log.
I presume you got that impression from the word "character".
I think in this context, it doesn't mean "glyph or letter", more like "a role on the scene".
And character 588 is exactly that video with external data.

**Comment by ROBERT-MCDOWELL** [comment]
ha ok understood, I learn everyday ;)... btw I reported the button component icon I set with setstyle() disappear when mouse goes over and rollout... I didn't see any error reported on this matter.
